docs/02.API-REFERENCE.md

```diff
@@ -1998,7 +1998,8 @@ handler (const jerry_value_t function_obj,
 
   // after this, script can invoke the native handler through "handler_field (1, 2, 3);"
   jerry_value_t prop_name = jerry_create_string ((const jerry_char_t *) "handler_field");
-  jerry_set_property (glob_obj, prop_name, func_val);
+  jerry_value_t set_result = jerry_set_property (glob_obj, prop_name, func_val);
```

Comment:
Add a comment for the short form (tell that if the result is not checked, it could be done in the following way).
LGTM after that.